test/author.spec.js

```diff
@@ -0,0 +1,26 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+
+const server = require('../server/index');
+
+const { apiURL } = global;
+
+chai.use(chaiHttp);
+const { expect } = chai;
+
+describe('Add Author', () => {
+  it('it should be able successfully add an author', (done) => {
```

Comment:
This probably should be reading `should add an author successfully`. No need for the `it`, and we should do our best to have clear grammar for other teammates to easily understand.